deployments/uservice-ghslacker/resources/ingress.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: uservice-ghslacker
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    certmanager.k8s.io/issuer: "api-lsst-codes-letsencrypt"
+spec:
+  tls:
+    - secretName: api-lsst-codes-ingress-tls
+      hosts:
+        - api-1.lsst.codes
```

Comment:
I _think_ that switching to a roundtable.lsst.codes host will be easier long-term, but I could be wrong. We could talk more about it at co-work.